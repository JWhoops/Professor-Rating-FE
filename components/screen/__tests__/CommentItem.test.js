import React from 'react';
import { render } from 'react-native-testing-library';

import Comment from '../CommentItem';

test('CommentItem renders all inputs as expected', () => {
  const { toJSON } = render(<Comment />);

  expect(toJSON()).toMatchSnapshot();
});