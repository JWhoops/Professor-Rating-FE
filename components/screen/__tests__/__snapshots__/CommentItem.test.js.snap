// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentItem renders all inputs as expected 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "borderRadius": 4,
      "height": "auto",
      "marginHorizontal": 8,
      "marginTop": 10,
      "padding": 20,
      "shadowColor": "rgba(96,96,96,1)",
      "shadowOffset": Object {
        "height": 4,
        "width": 0,
      },
      "shadowOpacity": 0.13,
      "shadowRadius": 4,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#666",
          "fontWeight": "bold",
        }
      }
    />
    <View
      style={
        Object {
          "position": "absolute",
          "right": -5,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "color": "#666",
        "marginTop": 12,
      }
    }
  />
</View>
`;
